docs/workflow/building/libraries/cross-building.md

```diff
@@ -87,6 +87,10 @@ You can also build just managed code with:
 
 The output is at `artifacts/bin/[BuildSettings]` where `BuildSettings` looks something like `net5.0-<TargetOS>-Debug-<Architecture>`. Ex: `artifacts/bin/net5.0-Linux-Debug-x64`. For more details on the build configurations see [project-guidelines](/docs/coding-guidelines/project-guidelines.md)
 
+Note that by default ILLinker trimming is enabled and libs built above for `arm` can't be used on other arches. To build architecture-independent libs use next command:
+
+    $ ./build.sh --arch arm /p:BuildNative=false /p:ILLinkTrimAssembly=false
```

Comment:
That command doesn't work anymore. I don't think that `BuildNative=false` is still a thing in the runtime world. Instead you would do the following, to just build managed code:

```
./build.sh libs.sfx+libs.oob+libs.pretest --arch arm /p:ILLinkTrimAssembly=false
```

Reply:
If you don't care about the testhost folder, you can even omit the `libs.pretest` subset. Same, if you just care about the shared framework libraries assemblies, you can just build the `libs.sfx` subset.